### ISSUE TITLE
Transmitter is shown as paired without pairing request and transmitter info

### ISSUE THREAD

**Issue by qwqatest** [Faifly/xDrip#250]
When pairing transmitter SN 81CC3D it is shown as paired after some time, but there wasn't a pairing request and transmitter info never showed up
Also, device name is displayed from another transmitter. I'm not sure, but I think that transmitter with name DexcomJD has never been paired to this phone
Could be related issue https://github.com/Faifly/xDrip/issues/206

![Screenshot at Nov 05 18-29-34](https://user-images.githubusercontent.com/47991168/98272785-24ef0180-1f9a-11eb-86b6-bd26a6c3b09c.png)

Probably there is some problem with 81CC3D but why it paired with the app in such weird way?
And today 81CC3D is paired successfully, but only with Mac.

Can be related to https://github.com/Faifly/xDrip/issues/247

**Comment by qwqatest** [comment]
Upd: I've tried on another device which has never been paired to any transmitter and device name is displayed from another transmitter that just lying around 

![Screenshot at Nov 06 16-44-39](https://user-images.githubusercontent.com/47991168/98379193-b2405d80-204f-11eb-8b5d-83ad3c8bc020.png)

**Comment by qwqatest** [comment]
UPD: Today I had the problem with pairing another transmitter - 80GYJD. It was a regular pairing after sensor insertion
During the first attempt, I saw this without a pairing request.

![Screenshot at Nov 09 16-34-17](https://user-images.githubusercontent.com/47991168/98554586-8e785400-22a9-11eb-91a6-4d94b6005d90.png)

After the second attempt, I got the pairing request but transmitter info was missing. I believe, I've tried to reopen "Transmitter" screen but info didn't appear

![Screenshot at Nov 09 16-34-44](https://user-images.githubusercontent.com/47991168/98554617-96d08f00-22a9-11eb-863f-24cfdddaa9ba.png)



Only after the third attempt pairing was successful. But still after pairing I had to reopen the "Transmitter" screen in order to see transmitter info (issue https://github.com/Faifly/xDrip/issues/267)
